.zuul.yaml

```diff
@@ -3,7 +3,67 @@
     name: packit-service/packit-service
     check:
       jobs:
-        - noop
+        - packit-service-pre-commit
+        - packit-service-tests
     gate:
       jobs:
-        - noop
+        - packit-service-pre-commit
+        - packit-service-tests
```

Comment:
I see you want to use secrets in pre-review jobs. For security reasons Zuul won't let you use secrets defined in a untrusted project. If the job was running is a post-review pipeline like the post pipeline it would have been fine, but with untrusted repositories Zuul reads and load the .zuul.yaml change from the PR then it could be easy to propose a PR with playbook changes and take advantages of the jobs privileges thanks to the secrets. So the secrets and playbooks, jobs that use the secrets should be added in a trusted (or config (this the same)) repository. Zuul does not read/loads the PR changes on the trusted repositories. Set up pre-review jobs to use secrets is not obvious so please read https://www.softwarefactory-project.io/zuul-hands-on-part-5-job-secrets.html it will give insight but it does not cover your use case.

For your use case you need to have the secrets, the playbooks and the jobs that use the secrets defined in a trusted (or config) repository. I advice you to start with a test job that only ensure you have access to the secret in the pre-run or run playbook. Let's create a repository packit-zuul-config and we will register it as a zuul trusted repo in sf-project.io as a starting point. I also need to check with @TristanCacqueray if we should create a Zuul packit tenant from our side.